src/Microsoft.Framework.Runtime/DependencyManagement/LockFileFormat.cs

```diff
@@ -71,7 +82,7 @@ private LockFile ReadLockFile(JObject cursor)
         {
             var lockFile = new LockFile();
             lockFile.Islocked = ReadBool(cursor, "locked", defaultValue: false);
-            lockFile.Version = ReadInt(cursor, "version", defaultValue: int.MaxValue);
+            lockFile.Version = ReadInt(cursor, "version", defaultValue: int.MinValue);
```

Comment:
`defaultValue` means this value should be used when there is no `version` property. Here we want to use `int.MinValue` when `version` property doesn't appear in lockfile.